experimental/custom-routing-with-envoy/README.md

```diff
@@ -0,0 +1,88 @@
+# Custom Routing With Envoy
+
+This example explains how you can use Signadot for realizing custom routing using envoy.
+
+## Set up
+
+To set up the demo application, you can run the following in your cluster:
+
+```
+kubectl create ns httpbin
+kubectl apply -f k8s/
+```
+
+This sets up [httpbin](https://httpbin.org/) in the cluster in the namespace
+httpbin. The httpbin application listens on port 80 and has an envoy sidecar
+that is running on port 8080 which is used for all communications in the
+cluster. 
+
+Once you have a signadot organization, connected a cluster and configured your
+signadot CLI, you can create a sandbox as shown below by running:
+
+```
+signadot sandbox apply -f ./sandbox-baseline.yaml 
```

Comment:
I would make the `cluster` a templated variable, and tell the user that he has to replace it with the corresponding value:

`signadot sandbox apply -f ./sandbox-baseline.yaml --set cluster=<your-cluster-name>`